Mathlib/NumberTheory/DiophantineApproximation.lean

```diff
@@ -369,22 +368,20 @@ theorem convergent_of_int {ξ : ℤ} (n : ℕ) : convergent ξ n = ξ := by
 #align real.convergent_of_int Real.convergent_of_int
 
 /-!
-Our `convergent`s agree with `GeneralizedContinuedFraction.convergents`.
+Our `convergent`s agree with `GenContFract.convs`.
 -/
 
 
-open GeneralizedContinuedFraction
+open GenContFract
 
-/-- The `n`th convergent of the `GeneralizedContinuedFraction.of ξ`
-agrees with `ξ.convergent n`. -/
-theorem continued_fraction_convergent_eq_convergent (ξ : ℝ) (n : ℕ) :
-    (GeneralizedContinuedFraction.of ξ).convergents n = ξ.convergent n := by
+/-- The `n`th convergent of the `GenContFract.of ξ` agrees with `ξ.convergent n`. -/
+theorem genContFract_conv_eq_convergent (ξ : ℝ) (n : ℕ) :
```

Comment:
Shouldn't this be
```suggestion
theorem genContFract_convs_eq_convergent (ξ : ℝ) (n : ℕ) :
```
to match the `convs`? Same comment for `exists_genContFract_conv_eq_rat`.

Reply:
I think according to the [naming convention doc](https://leanprover-community.github.io/contribute/naming.html#identifiers-and-theorem-names) (see the end of "Identifiers and theorem names"), if we include the namespace `genContFract`, it should be combined with the `convs` in lowerCamelCase. (I've edited the suggestion above accordingly.)

There's a case to be made that we could also leave out `genContFract` since `convs` is strictly speaking unambiguous, so I guess `convs_eq_convergent` might be OK as well, although maybe a bit harder to understand.

Reply:
The doc-string mentions `GenContFrac`, so just `convs` should be OK.

Reply:
I renamed.